targets/csharp/source/PlayFabSDK/source/PlayFabHttp/PlayFabPollyHttp.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Xbox.Publishing.PlayFabCSharpSDK.Polly.Interfaces;
+using PlayFab;
+using PlayFab.Internal;
+using Polly;
+using Polly.Wrap;
+
+namespace PlayFab.Internal
+{
+    /// <summary>
+    /// A Polly wrapped version of PlayFab Transport plug in which used
+    /// when making http requests to PlayFab Main Server.
+    /// </summary>
+    public class PlayFabPollyHttp : PlayFabHttp, IPollyTransportPlugin, IPlayFabPlugin
+    {
+        /// <summary>
+        /// Gets or set the name of the plug in. Used by the PluginManager when looking up
+        /// plugins upon request.
+        /// </summary>
+        public string Name;
+
+        /// <summary>
+        /// Gets the resilience policies defined.
+        /// </summary>
+        public AsyncPolicyWrap CommonResilience { get; private set; }
+
+        /// <summary>
+        /// Constructor for objects of type PollyTransportPlug.
+        /// <remarks>
+        /// Sets a default resilience policy with the fololwing common settings
+        /// 1) Sets the retry to 3 times and has an embedded backoff.
+        /// 2) > Sets a circuit breaker that will trigger is 25% of collapsed calls within 
+        ///      a 5 second window are failing.
+        ///    > Period for evaluation requires a burst of >=2RPS before evaluating breaker rule,
+        ///      requires a minimum of 10 requests in 5 seconds, and the circuit breaker will
+        ///      will be open for 15 second.
+        /// More information on the client can be found here: https://github.com/App-vNext/Polly
+        ///</remarks>
+        /// </summary>
+        public PlayFabPollyHttp()
+        {
+            var retryPolicy = Policy
+               .Handle<Exception>()
+                .Or<HttpRequestException>()
+                .Or<TaskCanceledException>()
+                .Or<TimeoutRejectedException>()
+                .WaitAndRetryAsync(3, retryAttempt => TimeSpan.FromSeconds(Math.Pow(2, retryAttempt)));
```

Comment:
Notes from meeting
Consider using retry header from playfab,
Consider using jitter
Review pf documentation 20 second breaker
Determine if we need a new interface